assets/section-password.css

```diff
@@ -167,14 +167,11 @@ body {
   margin-top: 0;
 }
 
-.password-main > section:only-child .newsletter__wrapper {
+.password-main > section:only-child .newsletter__wrapper:not(.email-signup-banner__box) {
```

Comment:
When is the `:not` selector here needed 🤔  
Doesn't it have a max-width anyway ? I might be missing a use case 😅